load_tests/complex.xml

```diff
@@ -0,0 +1,142 @@
+<?xml version="1.0"?>
+<!DOCTYPE tsung SYSTEM "/usr/local/share/tsung/tsung-1.0.dtd" [] >
+<tsung loglevel="notice">
+
+  <!-- Client side setup -->
+  <clients>
+    <client host="localhost" use_controller_vm="true" maxusers='30000'/>
+  </clients>
+
+  <!-- Server side setup -->
+  <servers>
+    <server host="www.ucsb-class-scheduler-token2.bdrmu4mt4q.us-west-2.elasticbeanstalk.com" port="80" type="tcp"></server>
+  </servers>
+
+
+  <load>
+    <arrivalphase phase="1" duration="30" unit="second">
```

Comment:
Maybe reduce to 10 or 20 seconds?

Reply:
Then I think it's gonna be hard to separate them in a run